### PR TITLE
Eurobot 2016 edition article

### PR DESCRIPTION
**Pull request by SyrianSpock**

### CODE REVIEW

File: coupe/2016.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+<div class="row">
+<div class="large-6 columns">
+<p>
+Goldorak is our smaller robot, named after the japanese anime series and manga, also known as Grandizer to English-speakers.
+</p>
+
+<p>
+It goes fishing in the water tank, sucking fish with its propeller and catching them with a magnet trap.
+Clamps on it ase allow it to collect seashells.
```
Comment (antoinealb) [inline coupe/2016.md]: Should be "base", no ?

File: coupe/2016.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+It is also responsible for plating the beach umbrella for the team in the end game.
```
Comment (antoinealb) [inline coupe/2016.md]: "planting"

File: coupe/2016.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+The main benefit of this technology is its huge versatility in performing the everchanging tasks each year in the competition.
+Each arm is brought to life by four geared DC motors manufactured by Faulhaber.
+At the end of each arm, a gripper made of five suction cups enables Debra to move, pick, and place game elements on the table.
+One of the novelties of this year is the addition of a fourth motorized axis: the continuous rotation of the gripper.
+Through this joint pass six electrical signals and four pneumatic tracks.
+
+The robots embed a wide range of actuators: geared DC motors, vacuum pumps, and servo motors: 13 for Goldorak and 31 for Debra.
+Each motor is connected to a homemade board dubbed the [motor board](https://github.com/cvra/motor-control-board).
+And to connected such a high number of elements, while keeping some flexibility in the design, a CAN bus was put in place.
+This allows un central node, containing the "intelligence" of the robot, to give instructions that will be executed locally by each child board.
```
Comment (antoinealb) [inline coupe/2016.md]: "one"

File: coupe/2016.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+The specialty of the club, that we iterate and improve upon since six years, is the use of articulated arms of SCARA type.
+The main benefit of this technology is its huge versatility in performing the everchanging tasks each year in the competition.
+Each arm is brought to life by four geared DC motors manufactured by Faulhaber.
+At the end of each arm, a gripper made of five suction cups enables Debra to move, pick, and place game elements on the table.
+One of the novelties of this year is the addition of a fourth motorized axis: the continuous rotation of the gripper.
+Through this joint pass six electrical signals and four pneumatic tracks.
+
+The robots embed a wide range of actuators: geared DC motors, vacuum pumps, and servo motors: 13 for Goldorak and 31 for Debra.
+Each motor is connected to a homemade board dubbed the [motor board](https://github.com/cvra/motor-control-board).
+And to connected such a high number of elements, while keeping some flexibility in the design, a CAN bus was put in place.
```
Comment (antoinealb) [inline coupe/2016.md]: "to connect"

**Review by antoinealb — CHANGES_REQUESTED** [review]
I would like to see the poster and game rules added to the download before merging this. Adding the CAD would be nice as well.

**Comment by SyrianSpock** [comment]
Thanks for catching the spelling mistakes.
The game rules are already lister under downloads.
I agree for the CAD files, but I don't have them.

**Comment by antoinealb** [comment]
ok, i will add the missing download later.

**Pull request merged**